CHANGELOG.md

```diff
@@ -2,19 +2,19 @@
 
 ## Version 3.4.4 GA
 
-Enhancements:
+# Enhancements:
```

Comment:
probably these should be `###` since they are sub-items of `## Version 3.4.4 GA`

Reply:
make sense.. # in the draft release probably is equivalent to ###